### ISSUE TITLE
Downloading file fails after reaching approximately 250MB

### ISSUE THREAD

**Issue by dm042023** [jlesage/docker-firefox#179]
When downloading a file larger than ~250MB the download fails.

**Comment by jlesage** [comment]
Please provide more details about your setup (the `docker run` command you used to create the container, compose file, etc).

I was able to download a 1GB file: https://speed.hetzner.de/1GB.bin
I created the container with `docker run --rm -p 5800:5800 jlesage/firefox`

**Comment by dm042023** [comment]
Thanks for the quick response!

I’ve tried a few variations. 

The default:

docker run -d 
    --name=firefox \
    -p 5800:5800 \
    -v /docker/appdata/firefox:/config:rw \
    jlesage/firefox

This didn’t work for me. The /docker directory is read-only. I’m installing this on a linux based network appliance. OM2200 from opengear. 



I created a new directory and ran this as both the root user and a new admin user:

docker run -d 
    --name=firefox \
    -p 5800:5800 \
    -v /etc/home/admin/firefoxappdata:/config:rw \
    jlesage/firefox



I read the instruction about possible problems with the user group and permissions. The last thing I ran was:

docker run -d --name=firefox -e USER_ID=0 -e GROUP_ID=0  -p 5800:5800 -v /etc/home/admin/firefoxappdata:/config:rw jlesage/firefox


root@om2216-l:/etc# id
uid=0(root) gid=0(root) groups=0(root),1001(admin)
